src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressKey);
+  }
+
+  pressKey = ((event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  });
```

Comment:
Don't use a function before declaration. Move it above life cycle methods.
Also please change the name, because for now `pressKey` and `pressedKey` look almost the same and can be easily confused.
```suggestion
  handlePressKey = ((event: KeyboardEvent) => {
    this.setState({ pressedKey: event.key });
  });
```

Reply:
I fixed it, thanks! But with function i had to add an 'eslint-disable-next-line'.
![image](https://user-images.githubusercontent.com/106486742/197407985-24f1e0db-ef24-46e5-bd3f-ae53a44d8ee5.png)